### DISCUSSION TITLE
How to disable the bluetooth controller

### DISCUSSION THREAD

**Discussion by elagil** [zephyrproject-rtos/zephyr#31377] [Q&A] [answered]
In an effort to save energy, which function is capable of disabling the bluetooth transceiver in the controller?

In other words, is there a counterpart to `bt_enable()`?

**Comment by RAlexeev** [comment, marked as answer]
It seems that about the same issue was already opened here https://github.com/zephyrproject-rtos/zephyr/issues/3192 and a kinda `bt_disable()` function (a counterpart to `bt_enable()`) is not implemented yet.
